setupext.py

```diff
@@ -368,6 +369,31 @@ def _compile(
     CCompiler.compile = _compile
 
 
+def get_python_include_dirs():
+    """Extracted from distutils.command.build_ext.build_ext#finalize_options(),
```

Comment:
?
```suggestion
    """Extracted from distutils.command.build_ext.build_ext.finalize_options(),
```

Reply:
Oh, that's just a convention I picked up from Java (I think?) to distinguish between free functions and object methods.

Reply:
I'm not sure it makes sense outside of Java. At least I never encountered this before and it just looks like a typo to my untrained eye.

Reply:
I'll get rid of it then.